packages/bpk-tokens/tokens/ios/base.react.native.es6.js

```diff
@@ -433,14 +449,6 @@ lineHeightXs,
 lineHeightXxl,
 };
 // Android only tokens
-export const borderRadiusLg = undefined;
```

Comment:
Same comment as in the test file, we probably need to keep these as they are Android only

Reply:
But they're not anymore. I'm reinstating them for iOS